pandas/core/reshape/reshape.py

```diff
@@ -1072,6 +1072,234 @@ def get_empty_frame(data) -> DataFrame:
         return DataFrame(dummy_mat, index=index, columns=dummy_cols)
 
 
+def from_dummies(
+    data: DataFrame,
+    to_series: bool = False,
+    prefix_sep: str | list[str] | dict[str, str] = "_",
+    columns: None | Index | list[str] = None,
+    dropped_first: None | str | list[str] | dict[str, str] = None,
+) -> Series | DataFrame:
```

Comment:
let's just always return a DataFrame, much simpler

Reply:
Good idea, and in line with [this perspective](https://github.com/pandas-dev/pandas/pull/41902#discussion_r650566136) as the `Series` can always be obtained ex post